project-reports/2024/2024-annual-Hyperledger-Indy.md

```diff
@@ -0,0 +1,98 @@
+
+Compared to 2022, the Pull Request count was down by about 25% to 189 for the year, while the number of contributors was slightly higher, up about 20% to 46. Compared to Q4 2023, the Q1 2024 report changes had the same ratios — down about 25% (to 24) in Pull Requests, up about 20% in contributors (to 13).
+
+While the existing implementation of Indy is extremely stable, and robust — for example, the Sovrin Foundation has been running its networks for 6 years, with 100% uptime since Feb. 2020 — there has been relatively little substantially new functionality added in some time. That may be changing as, in the latter half of 2023, the idea of “Indy-Besu” was raised and a successful PoC developed, demonstrating implementing Indy’s public-permissioned network model on top of a Besu network. Progress has already been good in 2024 with the Indy community agreeing on the creation of the [Indy-Besu repository](https://github.com/hyperledger/indy-besu) that further moves the implementation forward.
+
+The Hyperledger Indy ecosystem community came together at a series of Hyperledger Indy Ecosystem Summits (organized by the Sovrin Foundation), bringing together network operators, node operators, project maintainers, network users, vendors to discuss the state of Indy and proposals for moving the project forward. The summits were well-attended (up to 90 participants), a better understanding of the community of interest was gained, and next steps were proposed, with some (mostly notably, the Indy-Besu initiative) acted upon.
+
+## Maintainer Diversity
+
+_Guidance: How many maintainers do you have, and which organisations are they from? How has the maintainers and diversity of your maintainers changed in the past year? Has the number of active maintainers increased/decreased? Has the diversity of maintainers increased/decreased? Please include a link to your existing [MAINTAINERS file](https://github.com/hyperledger/toc/blob/gh-pages/guidelines/MAINTAINERS-guidelines.md) and the MAINTAINERS file from last year (if appropriate). This is a good opportunity to ensure that your MAINTAINERS file is up to date and to retire any maintainers._
+
+There are 35 individuals on 22 Indy GitHub Teams representing at least 16 organizations. We continue to have participation from (more or less) the same organizations as in 2022 — which is both good (still participating) and bad (few newcomers).
+
+From the last Quarterly Report: _Still needed -- a cleanup of the teams are needed as some of the individuals are no longer active in the community. No progress was made on that cleanup this quarter. We have the team lists, but have not queried those on the list to see if they are still interested in being maintainers._
+
+Typical Indy [Maintainers File](https://github.com/hyperledger/indy-plenum/blob/main/MAINTAINERS.md).
```

Comment:
Several maintainers' entries in the "Active Maintainers" table are incomplete, with no emails, Discord handles, and affiliations. Ideally, you should display this info so other users and potential contributors can reach out to them. I do see some of those maintainers in the Indy discord channels, so I don't believe there's a question of privacy here.